esphome/components/sensor/filter.cpp

```diff
@@ -237,7 +237,7 @@ optional<float> FilterOutValueFilter::new_value(float value) {
       return value;
   } else {
     int8_t accuracy = this->parent_->get_accuracy_decimals();
-    float accuracy_mult = pow10f(accuracy);
+    float accuracy_mult = pow(10.0, accuracy);
```

Comment:
```suggestion
    float accuracy_mult = powf(10.0f, accuracy);
```

Minor thing, but this will include `double` in our builds, which makes them a lot larger (at least when I wrote this, might have changed since then)